packages/website/lib/api.js

```diff
@@ -116,6 +116,20 @@ export async function createUnlimitedStorageRequest(authMethod, links, dataVolum
   );
 }
 
+export async function createEnterpriseTierRequest(authMethod, links, dataVolume, dataReadTypeAndFrequency, additionalInfo) {
+  await createUserRequest(
+    'StorageLimitBytes', // TODO: what should this be?
```

Comment:
It shouldn't be the same as the `createUnlimitedStorageRequest` IMO.

```suggestion
    'EnterpriseTierRequested',
```

Reply:
Sorry, yeah we talked about that earlier and I forgot to push the update

Reply:
Cool, I'm going to wind up ripping this out anyway for the email stuff.  So it's about to be completely different.

Reply:
OK, I've made a first attempt, but it looks like email is only available from the cron project which is probably going to be problematic.  I can try hoisting this into a shared project, if this cross project referencing isn't good (I don't feel good about it).